app/views/user/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1>Show users</h1>
```

Comment:
Please follow this [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png) for the user's show page.

Please create a method to show the user's 3 most recent posts and use it inside `app/views/users/show.html.erb`.